src/dbsync/tests/utils/string_helper_test.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Shouldn't this file be renamed to: "stringHelper_test.h" (the same for the .cpp one) ?